src/impute.ts

```diff
@@ -1,53 +1,22 @@
-import {Encoding} from './encoding';
-import {isFieldDef} from './fielddef';
-import {ImputeTransform} from './transform';
+import {ImputeMethods} from './vega.schema';
 
-export interface ImputeProperties {
-
-  // Should we add this?
-  // keyChannel?: string;
+export interface ImputeParams {
```

Comment:
Some of the properties here are shared with `ImputeTransform`, but are not shared in the code.  Please make sure they are shared by adding `ImputeBase` interface.  This way, any changes to the property docs would be always in sync for both the `ImputeParams` and `ImputeTransform`.

Reply:
Should I put `ImputeBase` in `src/transform.ts` or `src/impute.ts`?

Reply:
Let's say `impute.ts`